### PR TITLE
ASpIRE data preparation recipe

### PR DESCRIPTION
**Pull request by desh2608**
ASpIRE (LDC2017S21) recipe for both 'single' and 'multi' mic settings.

### CODE REVIEW

File: lhotse/recipes/aspire.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+        for k, segs in segments_grouped.items():
+            session, speaker = k
+            supervisions += [
+                SupervisionSegment(
+                    id=f"{session}-{speaker}-{i:03d}",
+                    recording_id=session,
+                    start=seg.start,
+                    duration=round(seg.end - seg.start, 4),
+                    speaker=speaker,
+                    text=seg.text,
```
Comment (pzelasko) [inline lhotse/recipes/aspire.py:159]: Can you add language=“English”?

**Review by pzelasko — COMMENTED** [review]
Thanks! Left one tiny comment.

File: lhotse/recipes/aspire.py (hunk 1 above)
Comment (desh2608) [reply lhotse/recipes/aspire.py:159]: Done!

**Comment by pzelasko** [comment]
Thanks!

**Pull request merged**